vault/README.md

```diff
@@ -6,22 +6,17 @@ A tool for secrets management, encryption as a service, and privileged access ma
 
 This directory contains a Kubernetes Helm chart to deploy a Vault server. For further details of how are we using Vault read this [post](https://banzaicloud.com/blog/oauth2-vault/).
 
-## Prerequisites Details
+**Requirements**
```

Comment:
What is the reason for replacing the headers with bold text?

Reply:
looked too flash while it only contained a single item, reduced the size H3